Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN dnf upgrade -y \
     && dnf clean all
 
 RUN groupadd -g 389 dirsrv \
-    && useradd -u 389 -g 389 \
+    && useradd -u 389 -g 389 -M \
         -c 'DS System User' \
```

Comment:
OTOH, this seems just bringing back the option that was lost in the proposed 612d18357a4f25c1ead8e9e0e5ccf1182c1ae5e4.